website/common/locales/en/defaultTasks.json

```diff
@@ -11,6 +11,20 @@
   "defaultTodoNotes": "You can either complete this To-Do, edit it, or remove it.",
 
   "defaultTodo1Text": "Join Habitica (Check me off!)",
+  "defaultTodo1Notes": "You can either complete this To-Do, edit it, or remove it.",
+
+  "defaultTodo2Text": "Learn About Habitica",
+  "defaultTodo2Notes": "Overview of New Users:\n\n💻 only: ☰(Menu - Top Bar) > Help\n\nFAQ:\n\n📱: ☰ > Help & FAQ\n\n💻 : ☰ > Help > FAQ",
```

Comment:
@MathWhiz Could you please show us a screenshot of what `defaultTodo2Notes` looks like when you hover over the task?